TypeDBConsole.java

```diff
@@ -275,16 +278,17 @@ private boolean transactionREPL(TypeDBClient client, String database, TypeDBSess
                 .terminal(terminal)
                 .variable(LineReader.HISTORY_FILE, TRANSACTION_HISTORY_FILE)
                 .build();
-        StringBuilder prompt = new StringBuilder(database + "::" + sessionType.name().toLowerCase() + "::" + transactionType.name().toLowerCase());
+        StringBuilder promptBuilder = new StringBuilder(database + "::" + sessionType.name().toLowerCase() + "::" + transactionType.name().toLowerCase());
         if (options.isCluster() && options.asCluster().readAnyReplica().isPresent() && options.asCluster().readAnyReplica().get())
-            prompt.append("[any-replica]");
-        prompt.append("> ");
+            promptBuilder.append("[any-replica]");
```

Comment:
can you fix this while we're here please? we use  { } if the `if` doesn't fit in one line